src/metabase/driver/sql_jdbc/sync/interface.clj

```diff
@@ -27,6 +27,13 @@
   driver/dispatch-on-initialized-driver
   :hierarchy #'driver/hierarchy)
 
+(defmulti have-any-schema-privileges?
```

Comment:
One more note: you should add `^{:added "0.39.0"}` metadata on driver multimethods so driver authors can see when stuff was introduced more easily

Reply:
Done (for the new multi).